test/integration/ClaimInTokens.t.sol

```diff
@@ -39,6 +39,12 @@ contract ClaimIntegrationTest is
             vm.stopPrank();
             csm.resume();
         }
+        if (!csm.publicRelease()) {
+            vm.startPrank(csm.getRoleMember(csm.DEFAULT_ADMIN_ROLE(), 0));
+            csm.grantRole(csm.MODULE_MANAGER_ROLE(), address(this));
+            vm.stopPrank();
+            csm.activatePublicRelease();
+        }
```

Comment:
could we just copy-paste 35-41 lines from `test/integration/DepositInTokens.t.sol` ?